src/components/DateSelector/DateSelectorPicker.tsx

```diff
@@ -42,6 +42,16 @@ export const DateSelectorPicker: FC<DateSelectorPickerProps> = ({
       initialMonth={date.toDate()}
       selectedDays={[date.toDate()]}
       onDayClick={(d) => onDayClick(dayjs(d))}
+      months={Array.from({ length: 12 }).map((_, i) =>
+        dayjs().month(i).format('MMMM')
+      )}
+      weekdaysLong={Array.from({ length: 7 }).map((_, i) =>
+        dayjs().day(i).format('dddd')
+      )}
+      weekdaysShort={Array.from({ length: 7 }).map((_, i) =>
+        dayjs().day(i).format('dd')
+      )}
+      firstDayOfWeek={1}
```

Comment:
Are you sure of this ?

Reply:
I reused the code I found in  `DayPicker` because it was working in French mode, if it is not correct please tell me why and i can correct both files

Reply:
I think this should be like this only when users use french mode, @yoannfleurydev Are you ok ?

Reply:
I thinks it ok (because here it's not to format a date but to populate months/weekdays in the right language)

Reply:
I don't understand @ivan-dalmet  because this PR changes the behavior 

**Before**

![image](https://user-images.githubusercontent.com/7570753/181498889-40863740-b741-4536-9b2c-2a0bde5dc4c9.png)

**After**

![image](https://user-images.githubusercontent.com/7570753/181499330-df08e933-666f-4b05-8719-78d831dd462f.png)

Reply:
Oh sorry I didn't see the firstDayOfWeek. But I still think that this is OK because the other components are setup with Monday as 1st day of week (and most of European countries start on Monday). Olso this is not based on the language but the country :/

Reply:
Should I change ? because as it depends on the country there is not much I can do